### PR TITLE
Remove a duplicated panel version history list

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

Before:
Screenshot 2024-09-12 at 14 53 41

After:
Screenshot 2024-09-12 at 14 31 44

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [x] tests executed by DN

### CODE REVIEW

File: scout/server/blueprints/panels/templates/panels/panels.html [hunk 1]
```diff
@@ -173,7 +173,7 @@
             <td>{{ panel.version }} ({{panel._id.generation_time.strftime('%Y-%m-%d')}} {% if panel.hidden %} deleted:{{ panel.date.date()}} {% endif %})</td>
             <td><span class="badge rounded-pill bg-secondary">{{ panel.genes|length}}</span></td>
             <td><button id="{{panel._id}}" type="button" data-bs-toggle="collapse" href="#paneldiv_{{panel._id}}" aria-expanded="false" aria-controls="paneldiv_{{panel._id}}" class="btn btn-primary btn-xs"><span class="fa fa-search-plus"></span></button><br>{{ history_view(panel._id, panel.panel_name) }}</td>
-            <td><button id="{{panel._id}}" type="button" data-bs-toggle="collapse" href="#modifydiv_{{panel._id}}" aria-expanded="false" aria-controls="modifydiv_{{panel._id}}" class="btn btn-warning btn-xs"><span class="fa fa-pen-square"></span></button><br>{{ history_view(panel._id, panel.panel_name) }}</td>
```
Comment (northwestwitch) [inline scout/server/blueprints/panels/templates/panels/panels.html:176]: 👍🏻

**Review by northwestwitch — COMMENTED** [review]
Nice! I was wondering why it looked so weird! 💯

File: scout/server/blueprints/panels/templates/panels/panels.html (hunk 1 above)
Comment (dnil) [reply scout/server/blueprints/panels/templates/panels/panels.html:176]: Yeah, I guess I have seen it so many times and not even wondered.. 😅

**Comment by northwestwitch** [comment]
Ah sorry, I've just realised I forgot to approve! 😆

**Review by northwestwitch — APPROVED** [review]
Yes, sorry!

**Comment by dnil** [comment]
No worries - there was no rush whatsoever with this.

**Pull request merged**